assets/elm/src/ListHelper.elm

```diff
@@ -0,0 +1,18 @@
+module ListHelper exposing (moveItemUp)
+
+import List.Extra exposing (findIndex, swapAt)
+
+moveItemUp : List a -> a -> Maybe (List a)
```

Comment:
I think this could `List a -> a -> List a`, if you defaulted to the original list if you got a Nothing.
Suggestion:
```
case findIndex (\current -> current == item) list of
  Just 0 -> list
  Just n -> list
             |> swap n (n-1)
             |> Maybe.withDefault list
  Nothing -> list
```